src/main/java/arona/MainWindow.java

```diff
@@ -0,0 +1,71 @@
+package arona;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * Controller for arona.MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Arona arona;
+    private Stage stage;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/Shiroko_Icon.png"));
+    private Image aronaImage = new Image(this.getClass().getResourceAsStream("/images/Arona_Icon.png"));
+
+    @FXML
+    public void initialize() {
+        scrollPane.vvalueProperty().bind(dialogContainer.heightProperty());
+    }
+
+    public void setArona(Arona arona) {
```

Comment:
I like how you followed the camelCase formatting, following the coding standard.